package.json

```diff
@@ -17,19 +27,13 @@
     "lint-staged": "^10.2.11",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.0.5",
+    "prettier-plugin-organize-imports": "^1.1.1",
```

Comment:
I didn't know that but looks good 👍 
https://github.com/simonhaenisch/prettier-plugin-organize-imports